### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by slitviachenko**
## Description
Updated README.md

## Related Issue
DEVX-2238

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ NPM-based tool to validate App Builder templates during submission process to [A
...
 
-| Field        | Description                                    | Required           |
-|--------------|------------------------------------------------|--------------------|
-| `categories` | Template must have categories defined          | :heavy_check_mark: |
-| `extension`  | Template must declare extension points it uses | :heavy_check_mark: |
-| `services`   | Template must declare services it uses         | :heavy_check_mark: |
+| Field        | Description                                                                                                | Required           |
+|--------------|------------------------------------------------------------------------------------------------------------|--------------------|
+| `categories` | Template must have categories defined                                                                      | :heavy_check_mark: |
+| `extension`  | Contains the extension metadata, like `serviceCode`.                                                       |                    |
```
Comment (lenaorobei) [inline README.md:17]: Suggestion:
Template might declare extension points it uses. Should contain the extension metadata, like `serviceCode`

File: README.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ NPM-based tool to validate App Builder templates during submission process to [A
 
 ## install.yml file
 
-| Field        | Description                                    | Required           |
```
[the remark this answers is not in the document]
Comment (slitviachenko) [reply README.md:14]: > Let's add type column here.

@lenaorobei , I am not sure what we might put there. For example, `services` are an array of objects that have their own structure, mandatory and optional fields, so `Array{code: string, ... <OPTIONAL PROPERTIES>}` won't help much. When we will have a validation schema for `install.yml`, we can point out to it.

File: README.md [hunk 3]
```diff
@@ -11,11 +11,11 @@ NPM-based tool to validate App Builder templates during submission process to [A
...
-|--------------|------------------------------------------------|--------------------|
-| `categories` | Template must have categories defined          | :heavy_check_mark: |
-| `extension`  | Template must declare extension points it uses | :heavy_check_mark: |
-| `services`   | Template must declare services it uses         | :heavy_check_mark: |
+| Field        | Description                                                                                                | Required           |
+|--------------|------------------------------------------------------------------------------------------------------------|--------------------|
+| `categories` | Template must have categories defined                                                                      | :heavy_check_mark: |
+| `extension`  | Contains the extension metadata, like `serviceCode`.                                                       |                    |
+| `services`   | An array of objects. Each object represents a service the template depends on. The `code` key is required. |                    |
```
[the remark this answers is not in the document]
Comment (slitviachenko) [reply README.md:18]: applied, thank you

File: README.md (hunk 1 above)
Comment (slitviachenko) [reply README.md:17]: applied, thank you

**Review by lenaorobei — APPROVED** [review]

**Pull request merged**